regret/_api.py

```diff
@@ -211,6 +212,54 @@ def __init_subclass__(Subclass, **kwargs):
 
         return deprecate
 
+    def module(
+        self,
+        version,
+        replacement=None,
+        removal_date=None,
+        addendum=None,
+    ):
+        """
+        Deprecate a module at import time.
+
+        Arguments:
```

Comment:
The  arguments are just copy/paste ... Maybe we can find a way to share the docstring.

Reply:
So partially I think what's here relates to the "policy object" concept, namely -- some projects do deprecation by version, some by time, some by both, some by neither -- so ultimately I think what will happen is `_Deprecator` will have a policy object that represents the policy for a given project, and then some of these arguments really belong passed to the policy object.

E.g. a `regret.policies.SemVer` object is the one that wants `version`, so potentially once that exists, the docstring for lots of methods here really will say they take `**kwargs` that get passed to the policy object, and not mention e.g. `version` directly.